src/main/java/frc/robot/OperatorBoard.java

```diff
@@ -0,0 +1,119 @@
+ */
+public class OperatorBoard {
+
+    private static Joystick HARDWARE;
+    public OperatorBoardButton BUTTON_ONE;
+    public OperatorBoardButton BUTTON_TWO;
+    public OperatorBoardButton BUTTON_THREE;
+    public OperatorBoardButton BUTTON_SIX;
+    public OperatorBoardButton BUTTON_ELEVEN;
+    public OperatorBoardButton BUTTON_TWELVE;
+    public OperatorBoardButton BUTTON_THIRTEEN;
+    public OperatorBoardButton BUTTON_FOURTEEN;
+
+    private static OperatorBoard instance;
+
+    /**
+     * The constructor is kept private so that I can make sure it is only run once,
+     * and that there is only ever one instance of OperatorBoard in existence.
+     */
+    public static OperatorBoard getInstance() {
+        if (instance == null) {
+            instance = new OperatorBoard();
+        }
+        return instance;
+    }
+
+    /**
+     * The constructor is kept private so that I can make sure it is only run once,
+     * and that there is only ever one instance of OperatorBoard in existence.
+     */
+    private OperatorBoard() {
+        HARDWARE = new Joystick(1);
+        BUTTON_ONE = new OperatorBoardButton(1);
+        BUTTON_TWO = new OperatorBoardButton(2);
+        BUTTON_THREE = new OperatorBoardButton(3);
+        BUTTON_SIX = new OperatorBoardButton(6);
+        BUTTON_ELEVEN = new OperatorBoardButton(11);
+        BUTTON_TWELVE = new OperatorBoardButton(12);
+        BUTTON_THIRTEEN = new OperatorBoardButton(13);
+        BUTTON_FOURTEEN = new OperatorBoardButton(14);
+    }
+
+    /**
+     * Access the x-axis of the left joystick.
+     * 
+     * @return Left is negative, right is positive.
+     */
+    private double getLeftJoystickX() {
+        return -HARDWARE.getRawAxis(0);
+        // This value is negated because something is weird with the wiring.
+    }
+
+    /**
+     * Access the y-axis of the left joystick.
+     * 
+     * @return Up is negative, down is positive.
+     */
+    private double getLeftJoystickY() {
+        return HARDWARE.getRawAxis(1);
+    }
+
+    /**
+     * Access the y-axis of the right joystick.
+     * 
+     * @return Up is negative, down is positive.
+     */
+    private double getRightJoystickX() {
+        return HARDWARE.getRawAxis(2);
+    }
+
+    /**
+     * Access the y-axis of the right joystick.
+     * 
+     * @return Up is negative, down is positive.
+     */
+    private double getRightJoystickY() {
+        return HARDWARE.getRawAxis(3);
+    }
+
+    public class OperatorBoardButton extends Button {
+        private int m_buttonNumber;
+
+        public OperatorBoardButton(int button) {
+            m_buttonNumber = button;
+        }
+
+        /**
+         * @return Whether or not the button is being pressed.
+         */
+        @Override
+        public boolean get() {
+            return HARDWARE.getRawButton(m_buttonNumber);
+        }
+
+        /**
+         * Turns the button's light on, and keeps it on until turnLightOff() is called.
```

Comment:
could use {@link turnLightOff()} for extra style

Reply:
also if you do that, if you rename turnLightOff(), it will also rename it in the javadoc

Reply:
Sounds cool

Reply:
it is extremely cool